vulnfeeds/cmd/cvelist2osv/version_extraction.go

```diff
@@ -242,17 +235,22 @@ func findInverseAffectedRanges(cveAff cves.Affected, cnaAssigner string) (ranges
 	var introduced []string
 	fixed := make([]string, 0, len(cveAff.Versions))
 	for _, vers := range cveAff.Versions {
+		versionValue := vers.Version
 		if vers.Status == "affected" {
-			introduced = append(introduced, vers.Version)
+			if len(strings.Split(versionValue, ".")) != 3 {
```

Comment:
Probably can be more specific to `== 2`. And if it's not == 2 or == 3, then just error out of this record.